data-sets-api-server/src/main/java/org/zowe/data/sets/services/zosmf/ListDataSetsAttributesZosmfRequestRunner.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Can you refactor ListDataSetAttributesZosmfRequestRunner & ListDataSetZosmfRequestRunner together into a new abstract class given their code is nearly identical, rather than have the copy-pasted code (this can possibly be done for the unit test as well)